EIPS/eip-5222.md

```diff
@@ -0,0 +1,252 @@
+---
+eip: 5222
```

Comment:
```suggestion
eip: 5218
```

Assigning a different unused EIP number, as 5222 is a bit too "nice"

Reply:
So "nice" EIP numbers are not allowed anymore? And editors get to decide whether a number is too nice to **actually** be used by an EIP? I know EIP-1 says that, but I'd argue that it wasn't simply so editors can decide whether a number is "too nice".